app/modules/users/saga/api/apiGet.test.js

```diff
@@ -130,4 +130,23 @@ describe(`apiGet`, (): void => {
     expect(_.last(result.allEffects).PUT.action.payload.error).toBeInstanceOf(UnsupportedOperationError);
   });
 
+  it(`sets its request status to FAILURE, when the api response does not contain a body`, async (): Promise<mixed> => {
+    const dummyAction = actions.apiGet(dummyId);
+    const dummyApiResponse = {
+      status: 200,
+      body: null,
+    };
+
+    const result = await expectSaga(sagas.apiGet, dummyAction)
+      .provide([
+        [select(platform.selectors.getUserAuth), null],
+        [call(api.users.get, dummyId, dummyToken), dummyApiResponse],
+      ])
+      .put(apiRequestsStatus.actions.setPending(a.API_GET))
+      .put.actionType(apiRequestsStatus.actions.setFailure(a.API_GET, new UnexpectedEmptyResponseError()).type)
+      .run();
+
+    expect(_.last(result.allEffects).PUT.action.payload.error).toBeInstanceOf(UnsupportedOperationError);
```

Comment:
That should be `.toBeInstanceOf(UnexpectedEmptyResponseError)`, after which the test will fail, because the provided `userAuth` is `null` and so the saga throws an `UnsupportedOperationError` instead of the expected `UnexpectedEmptyResponseError`.

(Side note: you can easily find this type of problem yourself by using `yarn run test-report modules/users` and seeing if your test has 100% coverage.)